src/modules/yutobot/index.js

```diff
@@ -75,6 +83,29 @@ const start = () => {
 						}
 						message.channel.send(owoifierEnabled ? "1" : "0");
 					}
+				
+				
+				case "Ⓐ":
+					//vars for time/date
+					ctx.textAlign = "center";
+					var dateObj = new Date()
+					var weekday = dateObj.toLocaleString("default", { weekday: "short" })
+					var today = new Date();
+					var dd = String(today.getDate()).padStart(2, '0');
+					var mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
+					const time = Date().slice(16,21);
+					today = dd + '/' +  mm
+						loadImage('./assets/wiimenu.png').then((image) => {
+						ctx.drawImage(image, 0, 0)
+					    const buffer = canvas.toBuffer("image/png");
+						ctx.fillStyle = "#828282";
+						ctx.fillText(weekday + " " + today, 980, 990);
+						ctx.fillStyle = "#9b9b9b";
+						ctx.fillText(time, 980, 860);
+						return message.channel.send(new Discord.MessageAttachment(canvas.toBuffer("image/png")))
+					}); 
+					break;
+					
```

Comment:
Put this also in the same new script and try exporting it as a function so that the main script here would just be a simple function call